pkg/managedcharts/cspadapter/cspadapter.go

```diff
@@ -19,7 +19,7 @@ const (
 	MLOChartName = "rancher-csp-adapter"
 	// PAYGChartNamespace is the namespace that we expect the adapter to be installed in,
 	// for the new Pay-As-You-Go (PAYG) license offering
-	PAYGChartNamespace = "cattle-csp-billing-adapter-system"
+	PAYGChartNamespace = "cattle-system"
```

Comment:
If it's going into cattle-system, you'll also need to add this to the system project here: https://github.com/rancher/rancher/blob/release/v2.7/pkg/settings/setting.go#L27

That also begs question, how does it impact backup/restore operator ? CSP-adapter is excluded, so does this follow same model ? https://github.com/rancher/backup-restore-operator/blob/914ffa31809250e2dda847625dca8389418ffbeb/charts/rancher-backup/files/default-resourceset-contents/rancher.yaml#L14

Reply:
Can you please elaborate? Per my understanding, the new PAYG CSP billing adapter will be maintained outside of Rancher. But I am not aware of addition requirements for putting it under the cattle-system namespace.